code/modules/mob/living/living.dm

```diff
@@ -88,7 +88,7 @@
 	var/they_can_move = TRUE
 	if(isliving(M))
 		var/mob/living/L = M
-		they_can_move = L.canmove //L.mobility_flags & MOBILITY_MOVE
+		they_can_move = L.mobility_flags & MOBILITY_MOVE
```

Comment:
This should definitely use CHECK_MOBILITY--heck if that didn't exist i'd ask for CHECK_BITFLAG instead